examples/aggregator/scripts/docker_push

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+readonly GLOBAL_SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")/../../../scripts/"
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=scripts/common
+source "${GLOBAL_SCRIPTS_DIR}/common"
+
+readonly AGGREGATOR_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator'
+readonly BACKEND_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator-backend'
+
+readonly IMAGES="$(find ./bazel-cache -name aggregator_image.tar)"
+readonly BACKEND_IMAGES="$(find ./bazel-cache -name aggregator_backend_image.tar)"
+if [[ -z ${IMAGES} || -z ${BACKEND_IMAGES} ]]; then
+  echo "Docker images have not been built, run './examples/aggregator/scripts/build_all' inside Oak Docker"
+  exit 1
+fi
+
+# Push Oak application server Docker image.
+docker load -i "./${IMAGES:1}"
```

Comment:
Why the `:1`?

Reply:
Deleted

Reply:
Is that not needed now? What if `IMAGES` returns more than one? I was just asking, not suggesting to remove it.

Reply:
I deleted it after checking that there is only one directory with this file.
Previously I had multiple - not I think I had them because I changed the output dir.
I think I'll just add a check+error that there a more that one `aggregator.tar` files, since there should only be one.